BUILD.md

```diff
@@ -367,13 +367,13 @@ See more information on the OpenVINO Execution Provider [here](./docs/execution_
 #### Build Instructions
 ##### Windows
 ```
-.\build.bat --config RelWithDebInfo --use_openvino <hardware_option>
+.\build.bat --config RelWithDebInfo --use_openvino <hardware_option> --build_shared_lib
```

Comment:
What happens now if we don't provide any of the extra options at all? does it build ORT + OV EP together as it did in the past?

Reply:
No it does not manohar

Reply:
@sfatimar I just tried to build ORT + OV EP **without --build_shared_lib** flag. It got built successfully and I was able to test out onnx_test_runner, onnxruntime_perf_test, onnx_backend_test_series.py and python sample Application as well.

**command I used:**
./build.sh --config Debug --use_openvino CPU_FP32 --build_wheel --parallel